### ISSUE TITLE
Cluster slow: FEAT & mrtrix not working

### ISSUE THREAD

**Issue by Lmli** [c3nl-neuraldynamics/c3nl_cluster#48]
Hi Cluster Team,

Sorry to be the bringer of bad news, but there's a couple of issues:

1) I'm running tckgen from mrtrix on a qlogin - it was working Tuesday evening (overnight) but has now stopped running (no new outputs in about 12 hours)

2) I started several 1st level FEATS, which have all started (and registration steps all run) but have now stopped (and failed) and the stats step (which is where they were failing before).

I'm not sure whether it's an issue of asking too much of the cluster again? A timing thing? A parallelisation issue?

Please help!
Thank you,
Lucia

**Comment by petehellyer** [comment]
Can you point me towards some examples please :)

P

**Comment by petehellyer** [comment]
Also, Eyal. Can you do an rsync update on /apps from true old copy. I suspect some changes I made to fsl_sub a few weeks ago aren’t included in the new location.

**Comment by Lmli** [comment]
the feats being run are in this location: /home/lli/PATIENTS/PRE/SST/PPI/1stlevel
(script in use: /home/lli/scripts/PPI_1stlevel.sh)

the mrtrix outputs are in this location: /home/lli/tools/MRTRIX/subj/
(script in use: /home/lli/scripts/mrtrix.sh)

**Comment by eyalsoreq** [comment]
rsyncing  fsl folder now

**Comment by petehellyer** [comment]
Coolio! That should fix current FEAT problems.

**Comment by Lmli** [comment]
Hello!

I'm so sorry, but there appears to be an issue with first level FEATs again.

I ran a whole load last night, which showed "Dependency Never Satisfied" this morning. I killed all my jobs and started a few again. These also seem to have stopped now... I don't know if it's because there's lots going on at the moment, or whether it's the old FEAT issue?

The feats I am attempting to run are in the following locations:
/home/lli/CONTROLS/SST/PPI/July2017/1stlevel/forCvP/
/home/lli/PATIENTS/PRE/SST/PPI/1stlevel/ - *RIFGai.feat (the other feats were run on my local machine which I uploaded in to this folder)

Thank you!
Lucia

**Comment by petehellyer** [comment]
this should now be fixed, let me know if there are any problems

**Comment by Lmli** [comment]
Hi Pete, so sorry to say that this still appears to be an issue.

I have run a feat (test+.feat) in: /home/lli/PATIENTS/PRE/SST/postFIX/1stlevel/
I started it yesterday (after you sent an email saying Cluster was back up) and it's, i imagine, timed out. The jobs associated with this feat have "Dependency Never satisfied" again.

Thank you,
Lucia

**Comment by petehellyer** [comment]
*smashes head against wall*

Ok, I've made some changes, give it a half hour and it should work again.

**Comment by Lmli** [comment]
Thanks Pete! I hope you didn't hurt your head against the wall....?!

I'm getting on a plane in a bit so I'll next give it a go when I land just to give it plenty of time.

**Comment by Lmli** [comment]
Hi,

Sorry to disturb again... I'm running into problems running higher level FEATs now, which has never happened before :(
I've attempted to rerun several times (thinking maybe cluster was just being a bit slow), but nothing has successfully run for 4 days....

The error by the job ID says "launch failed requeued held" and the report.html show that they're "still running" but this is several days on. Is there a time-out issue/permissions issue going on?

An example of group FEATs in this state can be found in:
/home/lli/PATIENTS/PRE/SST/postFIX/3rdlevel/

ANODALttest_LvHcon.gfeat
CATHODALttest_LvHcon.gfeat
SHAMttest_LvHcon.gfeat

Thank you,
Lucia

**Comment by Lmli** [comment]
Higher level FEATS works now.

Not tried MRTRIX again.

**Comment by inesviolante** [comment]
Hello,
I've tried FIX and I've replicated Lucia's error:

Error in file(file, "rt") : cannot open the connection
Calls: read.csv -> read.table -> file

This seems to be because FSL is not in the startup.m file in Matlab.
I don't have permissions to add a file to the Matlab path. Is it possible to create a startup file with this:

setenv( 'FSLDIR', '/apps/software/fsl/5.0.10');
fsldir = getenv('FSLDIR');
fsldirmpath = sprintf('%s/etc/matlab',fsldir);
path(path, fsldirmpath);
clear fsldir fsldirmpath;


Thank you!
